### PR TITLE
fix: Use correct rules for `ephemeralcontainers`

### PR DESCRIPTION
**Pull request by viccuad**
## Description

Relates to https://github.com/kubewarden/allowed-proc-mount-types-psp-policy/issues/57

Set correct rules for `ephemeralcontainers`.
Prepare for release.

## Test

CI

### CODE REVIEW

**Review by jvanz — APPROVED** [review]

**Review by flavio — CHANGES_REQUESTED** [review]

File: artifacthub-pkg.yml [hunk 1]
```diff
@@ -76,6 +76,6 @@ annotations:
       - v1
       resources:
       - pods
-      - pods/ephemeralcontainers
+      - ephemeralcontainers
```
[the remark this answers is not in the document]
Comment (viccuad) [reply artifacthub-pkg.yml]: Well spotted. I have amended the PR to drop `ephemeralcontainers`, will keep https://github.com/kubewarden/allowed-proc-mount-types-psp-policy/issues/57 open to work on audit-scanner.

[the remark this answers is not in the document]
Comment (flavio) [reply artifacthub-pkg.yml]: Yeah, I think for audit-scanner we have to handle the case of a policy that is targeting **only** a sub-resource. I don't think this is something common, we could just track it and work on that when someone complains about that

**Review by flavio — APPROVED** [review]

File: artifacthub-pkg.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (viccuad) [reply artifacthub-pkg.yml]: In my opinion the audit-scanner should we working seamlessly already; I don't think it sees a difference between a resource or a subresource. Hence why the policy was already being triggered by the audit-scanner.

Nevertheless, I opened https://github.com/kubewarden/audit-scanner/issues/322 for bookeeping.

**Comment by viccuad** [comment]
tagged in main as 0.1.10.

**Pull request merged**